snap/local/build_and_install.sh

```diff
@@ -1,14 +1,50 @@
 #!/bin/bash
+# Cross-compile the Certbot snap from local sources for the specified architecture,
+# and install it if this architecture is also the the current machine one.
+# This script is designed for CI tests purpose.
+# Usage: build_and_install.sh [amd64,arm64,armhf]
```

Comment:
It is totally possible. Given that in fact `i386` is used more than `aarch64` (while I think it will change soon with the new Raspberry 8Go ...), I agree it is a good idea to add it.

Reply:
Done.